src/powershell-win-core/Microsoft.PowerShell_profile.ps1

```diff
@@ -0,0 +1,10 @@
+# Default profile for all users for PowerShell host
+
+if ($IsWindows)
+{
+	# Add Windows PowerShell PSModulePath to make it easier to discover potentially compatible PowerShell modules
+	# If a Windows PowerShell module works or not, please provide feedback at https://github.com/PowerShell/PowerShell/issues/4062
+
+	Write-Warning "Appended Windows PowerShell PSModulePath"
+	$env:psmodulepath += ";${env:userprofile}\Documents\WindowsPowerShell\Modules;${env:programfiles}\WindowsPowerShell\Modules;${env:windir}\system32\WindowsPowerShell\v1.0\Modules\"

```

Comment:
This only gets the default, if the Windows PSModulePath is modified by installing Azure cmdlets or SQL, then they won't be available. We would need something like this:
```Powershell
$env:PSModulePath += (";" + (& ${env:windir}\system32\windowspowershell\v1.0\powershell.exe '$env:PSModulePath'))
```